src/components/Cluster/ClusterDetail/NodePool.js

```diff
@@ -93,6 +93,22 @@ class NodePool extends Component {
     this.scaleNodePoolModal.setNodePool(nodePool);
   };
 
+  formatInstanceDistribution = () => {
+    const { instance_distribution } = this.props.nodePool.node_spec.aws;
+
+    const baseCapacity = instance_distribution?.on_demand_base_capacity ?? '-';
+    let spotInstancePercentage =
+      instance_distribution?.on_demand_percentage_above_base_capacity ?? '-';
+
+    /* eslint-disable-next-line no-magic-numbers */
+    spotInstancePercentage =
+      spotInstancePercentage !== '-'
+        ? `${100 - spotInstancePercentage} %`
+        : '-';
```

Comment:
Apologize for the nitpicks 😅 

```suggestion
if (spotInstancePercentage !== '-') {
  spotInstancePercentage = `${100 - spotInstancePercentage} %`;
}
```